### PR TITLE
Translation to Portuguese -  p02_ch07_02_more_challenging_tests_of_up…

### PR DESCRIPTION
**Pull request by nunojesus**
…s.md

Update p02_ch07_02_more_challenging_tests_of_ups.md - Translation to Portuguese

Hello,
I translate 1112 words to Portuguese, so I finish this file. 

I think my translation is good, so if you can merge it please !!!

Greetings Nuno Jesus

### CODE REVIEW

File: pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
+### "Não Comas Peixe"
 
-What would be the status of the moral proposition: “It is evil to eat fish”?
+Qual seria o status da proposição moral: "é mau comer peixe"?
 
-Clearly, this proposition seems to satisfy at least some of the requirements of UPB – it appears universal, independent of time and place, and relatively objective.
+Claramente, esta proposição parece satisfazer pelo menos alguns dos requisitos do UPB - aparenta ser universal, independente de tempo e lugar, e relativamente objetiva.
 
-Yet it seems hard for us to reasonably call this a truly *moral* theory – why?
+No entanto, parece razoalmente difícil para nós chamar isto de uma verdadeira teoria *moral* - porque?
```
Comment (swfsql) [inline pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: "chamar" has the wrong conjugation; "porque" is a question, no answer, and in the end of the phrase, followed by interrogation, should be "por quê"

File: pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
 
-First of all, “evil” encompasses actions that can be prevented through the use of force. Rape is “evil,” and so I can use force to defend myself against someone attempting to rape me.
+Primeiramente, o "mau" engloba ações que podem ser prevenidas através do uso da força. Violação é "mau," e então eu posso usar a força para me defender de alguém que me está a tentar violar.
 
-Can I justly shoot someone who eats a piece of fish?
+Poderia eu justamente dar um tiro em alguém que comesse um pedaço de peixe?
 
-It would seem silly to argue that I can – but why?
+Parece uma tolice argumentar que eu poderia - mas porque?
```
Comment (swfsql) [inline pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: same previous "porque"

File: pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
 
-Also, the word “fish” remains problematic in the formulation, since it is too specific to be universal. The proper UPB reformulation is: “it is evil for people to kill living organisms.”
+Também, a palavra "peixe" continua a ser a problemática na formulação, já que é muito específica para ser universal. A reformulação apropriada do UPB é: "é mau para pessoas matarem outros organismos seres vivos."  
 
-If, however, it is evil to kill, we again face the problem of “inevitable evil.” No human being can exist without killing other organisms such as viruses, plants, or perhaps animals. Thus “human life” is defined as “evil.” But if human life is defined as evil, then it cannot *be* evil, since avoidance becomes impossible.
+Se, no entanto, é mau matar,  temos que encarar novamente o problema de "mau inevitável." Nenhum ser humano pode existir sem matar outros organismos como vírus, plantas ou talvez animais. Assim, "a vida humana" é definida como "mau." Mas se vida humana é definida como mau, então ela na pode *ser* má, já que evitabilidade se torna impossível.
```
Comment (swfsql) [inline pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: There is a small typo, "então ela na pode" should be "então ela não pode"

File: pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md (hunk 1 above)
Comment (nunojesus) [reply pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: Thank you for the correction.

File: pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md (hunk 2 above)
Comment (nunojesus) [reply pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: Thank you for the correction.

File: pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md (hunk 3 above)
Comment (nunojesus) [reply pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: Thank you for the correction.

Comment (swfsql) [reply pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: I think there is a way for you to do the corrections, @nunojesus

Comment (nunojesus) [reply pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: How can I do that?

Comment (nunojesus) [reply pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: I thought that the utopian moderators would review the translations and change them if necessary.

Comment (romero350) [reply pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md]: You can commit on the same branch on your fork and the PR will update itself

**Pull request merged**